assets/src/common/components/select-media-frame.js

```diff
@@ -0,0 +1,182 @@
+/**
```

Comment:
Much of this is simply copied and slightly refactored from [featured-image-select-media-frame.js](https://github.com/ampproject/amp-wp/pull/2737/files#diff-094dcdc5d37691ad94173f75bc4df3f0).

Reply:
The name is changed because it doesn't only apply to featured images now, it also applies enforces the file type.